src/components/Dropdown/taxonomyDropdown/DropdownTag.jsx

```diff
@@ -81,84 +31,38 @@ class DropdownTag extends Component {
-    const { handlePopupClick, tagProperties, tag } = this.props;
-    const { value } = e.target;
+  handleSetTagProperty(id) {
+    const { handlePopupClick, tag } = this.props;
 
-    const tagProperty = tagProperties.find(tagProp => tagProp.id === value);
-    if (tagProperty) {
-      this.setState({ tagProperty });
-      handlePopupClick({ ...tag, relevanceId: tagProperty.id });
+    if (id) {
+      handlePopupClick({ ...tag, relevanceId: id });
     }
   }
 
-  toggleHighligth() {
-    this.setState(prevState => ({
-      isHighlighted: !prevState.isHighlighted,
-    }));
-  }
-
   render() {
-    const { tag, tagProperty, isHighlighted } = this.state;
-    const { messages, name, tagProperties } = this.props;
-
-    let tagRelevances;
-    if (tagProperties) {
-      tagRelevances = tagProperties.map(
-        property =>
-          property.name ? (
-            <DropdownTagPropertyItem
-              key={property.id}
-              itemProperty={tagProperty}
-              tagProperty={property}
-              handleSetTagProperty={this.handleSetTagProperty}
-            />
-          ) : (
-            ''
-          ),
-      );
-    }
-    const tagShortName = tagProperty.name
-      ? tagProperty.name.charAt(0).toUpperCase()
-      : '';
-
-    const tagPropertyItem = <strong>{tagShortName}</strong>;
+    const { messages, name, tagProperties, tag } = this.props;
 
-    const filterTooltipItem = (
-      <div {...tagClasses('radio')} tabIndex={-1} role="radiogroup">
-        <div {...tagClasses('radio', 'description')}>
-          {messages.toolTipDescription}
-        </div>
-        {tagRelevances}
-      </div>
-    );
-
-    const filterItem = (
-      <ToolTip
-        name={name}
-        direction="right"
-        messages={{ ariaLabel: 'tooltip' }}
-        content={filterTooltipItem}>
-        {tagPropertyItem}
-      </ToolTip>
-    );
-
-    let tagItem;
-    if (tag.primary) {
-      tagItem = tagPropertyItem;
-    } else if (tagProperty && name === 'filter') {
-      tagItem = filterItem;
-    }
+    const isFilterTag = name === 'filter';
+    const isTopicTag = name === 'topics';
+    const isPrimaryTopic = !!tag.primary;
 
     return (
       <div
-        role="button"
-        tabIndex="0"
-        onClick={this.onClick}
-        onKeyPress={this.onClick}
-        {...tagClasses('', isHighlighted ? 'highlighted' : '')}>
+        {...tagClasses('', isTopicTag && !isPrimaryTopic && 'shouldHaveHover')}>
         <div {...tagClasses('description')}>{tag.name}</div>
-        {tagItem && <div {...tagClasses('item')}>{tagItem}</div>}
+        {isFilterTag && (
+          <FilterRelevanceSelector
+            currentRelevance={tag.relevanceId}
+            messages={messages}
+            tagProperties={tagProperties}
+            handleSetTagProperty={this.handleSetTagProperty}
+          />
+        )}
+        {isPrimaryTopic && (
+          <div {...tagClasses('item')}>
+            <strong>P</strong>
```

Comment:
What is this?

Reply:
Hva tenker du på, strong P? Det er valgt emne som er primær